src/NixFromNpm/Conversion/ToDisk.hs

```diff
@@ -174,16 +175,24 @@ initializeOutput = do
     [] -> do -- Then we are creating a new root.
       unlessExists defaultNixPath $
         writeNix (outputPath </> "default.nix") $ rootDefaultNix npm3
-      createDirectoryIfMissing (outputPath </> "nodeLib")
+
+      -- Get the path to the files bundled with nixfromnpm which
+      -- contain nix libraries.
+      nixlibs <- getDataFileName "nix-libs"
+
+      let inputNodeLib  = nixlibs    </> "nodeLib"
+      let outputNodeLib = outputPath </> "nodeLib"
+
       putStrsLn ["Generating node libraries in ", pathToText outputPath]
-      -- Get the path to the files bundled with nixfromnpm which contain
-      -- nix libraries.
-      nodeLibPath <- (</> "nodeLib") <$> getDataFileName "nix-libs"
-      -- Copy each of these into the target (unless they exist).
-      forItemsInDir_ nodeLibPath $ \path -> do
-        whenM (isFile path) $ do
-          unlessExists (outputPath </> "nodeLib" </> filename path) $
-            copyFile path (outputPath </> "nodeLib" </> filename path)
+
+      shelly $ do
+        rm_rf outputNodeLib
+        cp_r inputNodeLib outputNodeLib
+
+        let tools     = outputNodeLib </> "tools"
+        let toolsText = either id id (toText tools)
```

Comment:
have a function `pathToText` which does the same thing. It should already be in scope